Sources/SubviewDistributor.swift

```diff
@@ -175,18 +175,18 @@ public struct SubviewDistributor {
         // Calculate the ViewPosition (e.g. the Anchor) on `superview` for aligning. Make sure to take the `rect` into account.
         
         var leadingViewPosition: ViewPosition = {
-            let leftHorizontalInset = distributionRect.minX
-            let rightHorizontalInset = (superview.bounds.maxX - distributionRect.maxX)
-            let topVeritcalInset = distributionRect.minY
-            let bottomVerticalInset = (superview.bounds.maxY - distributionRect.maxY)
-            
+            let leftHorizontalOffset = distributionRect.minX
+            let horizontalMidpoint = distributionRect.midX - leftHorizontalOffset
+            let topVerticalOffset = distributionRect.minY
+            let verticalMidpoint = distributionRect.midY - topVerticalOffset
+
             switch direction {
             case .vertical:
-                return superview.top + UIOffset(horizontal: (leftHorizontalInset - rightHorizontalInset) / 2.0,
```

Comment:
The previous flow here increased the chances of a rounding error by rounding both the `superview`'s midpoint and the `distributionRect`'s midpoint. Now we're just rounding the midpoint on the distributionRect.